### PR TITLE
Servo function upgrade.

### PR DESCRIPTION
**Pull request by veRYANgry**

### CODE REVIEW

File: SpiderBot.c [hunk 1]
```diff
@@ -2,25 +2,20 @@
...
 */
 
-#include "simpletools.h"                      // Include simpletools header
+#include "simpletools.h"                  
 
-i2c *eeBus;                                   // I2C bus ID
+i2c *eeBus;
 void setupServoController(const i2c *eeBus);
-void setServo(const i2c *eeBus, long start, long stop);
+void setServoAmount(const i2c *eeBus, uint16_t rotation, uint8_t location);
```
Comment (Jdban) [inline SpiderBot.c:9]: I would suggest a more descriptive name

File: SpiderBot.c [hunk 2]
```diff
@@ -29,25 +24,36 @@ int main()
...
 
-void setServo(const i2c *eeBus, long start, long stop){
-  i2c_out(eeBus, 0x40, 0x06, 1, &start, sizeof(start));  // On pwm.
-  i2c_out(eeBus, 0x40, 0x08, 1, &stop, sizeof(stop));  // Off pwm.  
-}  
-
-void setServo(const i2c *eeBus, long start, long stop){
-  i2c_out(eeBus, 0x40, 0x06, 1, &start, sizeof(start));  // On pwm.
-  i2c_out(eeBus, 0x40, 0x08, 1, &stop, sizeof(stop));  // Off pwm.  
+// Set the servo from 0 to 180 degrees.
```
Comment (Jdban) [inline SpiderBot.c]: I feel like comments for method description should be more javadocs style

/*
 *
 */

**Review by Jdban — CHANGES_REQUESTED** [review]
Autoformat yo code pls. Spacing is terrible

File: SpiderBot.c (hunk 2 above)
Comment (veRYANgry) [reply SpiderBot.c]: Will add using the following style for doxygen:
/** Brief description which ends at this dot. Details follow
 *  here.
 */

**Comment by Jdban** [comment]
Merge conflict somehow btw :P

**Pull request closed without merging**